### ISSUE TITLE
Use rtt_log with rtt_target

### ISSUE THREAD

**Issue by gmmyung** [sourcebox/rtt-log-rs#1]
This linker error shows up when trying to use rtt_target with rtt_log. Is there a way to use regular `rprintln` with rtt-log?
```
  = note: rust-lld: error: duplicate symbol: _SEGGER_RTT
          >>> defined at eerie_embedded.7c2001f6d11f5a70-cgu.0
          >>>            /Users/gmmyung/Developer/eerie-embedded/target/thumbv7em-none-eabihf/release
/deps/eerie_embedded-b5a2d1bcc30af02c.eerie_embedded.7c2001f6d11f5a70-cgu.0.rcgu.o:(_SEGGER_RTT)
          >>> defined at rtt_log.c37f8a5fd759f351-cgu.0
          >>>            rtt_log-ade76af0363c4fae.rtt_log.c37f8a5fd759f351-cgu.0.rcgu.o:(.bss._SEGGER
_RTT+0x0) in archive /Users/gmmyung/Developer/eerie-embedded/target/thumbv7em-none-eabihf/release/dep
s/librtt_log-ade76af0363c4fae.rlib
```

**Comment by sourcebox** [comment]
Thanks for your report. The issue is caused by the fact that RTT requires a global symbol `_SEGGER_RTT` to be included in the binary. As this crate is based on `rtt-target`, which includes this symbol, adding `rtt-target` as an additional dependency, the symbol is included twice and the linker error is thrown.

I just pushed an update that re-exports `rtt-target`, so you don't need to include it separately. Please check out the current git version and drop me a note if it works. I will then publish a 0.3 version on crates.io.

**Comment by gmmyung** [comment]
Yup, it works for me, but I moved on just copy-pasting snippets of `rtt-log` for my needs. I think moving the `rtt_init` part to the user, and then initializing the logger by users attaching rtt channels makes more sense. However, please note that this is just my opinion, so don't take it too seriously.

**Comment by sourcebox** [comment]
There was some discussion a while ago to `rtt-target` directly integrate the log functionality. The developers agreed on it, it's simply not done. This would eliminate the need for `rtt-log` completely, if fact the only reason I created it was to emulate that missing functionality. So I hope that it will be implemented some day, allowing more complex setups and version conflicts.

**Comment by sourcebox** [comment]
Closed via 5d514bc80f534831611722b4d880ab2b648e4f76
